Traceback (most recent call last):
  File "train_pointnet_with_naive_view_rump.py", line 124, in <module>
    main()
  File "train_pointnet_with_naive_view_rump.py", line 121, in main
    train(view)
  File "train_pointnet_with_naive_view_rump.py", line 110, in train
    train_loss = train_model(model, dataloader['train'], criterion, device, optimizer, lr_scheduler)
  File "train_pointnet_with_naive_view_rump.py", line 21, in train_model
    outputs = model(inputs)
  File "E:\Programs\miniconda\envs\skeypoint\lib\site-packages\torch\nn\modules\module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "E:\ZOOTOS RnD\NIAS\SMART FARM\DairyCow\Paper2\baselines\..\Pnet\pointnet.py", line 164, in forward
    x1, trans, trans_feat = self.feat(x)
  File "E:\Programs\miniconda\envs\skeypoint\lib\site-packages\torch\nn\modules\module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "E:\ZOOTOS RnD\NIAS\SMART FARM\DairyCow\Paper2\baselines\..\Pnet\pointnet.py", line 120, in forward
    x = self.bn3(self.conv3(x))
  File "E:\Programs\miniconda\envs\skeypoint\lib\site-packages\torch\nn\modules\module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "E:\Programs\miniconda\envs\skeypoint\lib\site-packages\torch\nn\modules\conv.py", line 313, in forward
    return self._conv_forward(input, self.weight, self.bias)
  File "E:\Programs\miniconda\envs\skeypoint\lib\site-packages\torch\nn\modules\conv.py", line 309, in _conv_forward
    return F.conv1d(input, weight, bias, self.stride,
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 6.10 GiB (GPU 0; 24.00 GiB total capacity; 16.83 GiB already allocated; 941.00 MiB free; 21.79 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF